x/farming/keeper/grpc_query.go

```diff
@@ -242,6 +323,46 @@ func (k Querier) Rewards(c context.Context, req *types.QueryRewardsRequest) (*ty
 	return resp, nil
 }
 
+// UnharvestedRewards queries all unharvested rewards for the farmer.
+func (k Querier) UnharvestedRewards(c context.Context, req *types.QueryUnharvestedRewardsRequest) (*types.QueryUnharvestedRewardsResponse, error) {
```

Comment:
Could you update [Demo script](https://github.com/cosmosquad-labs/squad/blob/5d4b18bad291c114b01057c019f630c1a34e56ae/docs/demo/farming_public_plans.md) for using this query and fixed advanceEpoch, other queries?

Reply:
I updated it.

Reply:
Could you update https://github.com/cosmosquad-labs/squad/blob/5d4b18bad291c114b01057c019f630c1a34e56ae/docs/demo/farming_public_plans.md also?